### PR TITLE
feat: Finish hero section + app bar

### PR DESCRIPTION
**Pull request by doanhtu07**
# Status

- [x] Finish with UI
- [x] The app bar buttons will need to link to the sections or pages when they are finished
- [x] Where is the app bar on mobile view? Figma does not have it
- [x] Review https://github.com/acmutd/hackportal/pull/275
- [ ] The app bar icons are not finalized
- [ ] Still missing sign out option on app bar

**NOTE:** I've played around with the app bar scrolling effect. Let me know what you guys think.

# Showcase

## Navbar

https://github.com/acmutd/hackportal/assets/55064969/384f61e0-b539-4284-9222-f40ffdea708c

## Floating particle

If we don't want this, I can simply remove the transition. This floating effect is for comment from @uy-seng. Instead of scrolling and transitioning, I play around with the floating effect first.

We can discuss more about this.

https://github.com/acmutd/hackportal/assets/55064969/d211988a-f62b-4efc-9de7-d6c59ebe868c

### CODE REVIEW

**Comment by uy-seng** [comment]
we can link the position of circle relative to the scroll to animate the circle similar to figma, cuz in the figma the circle move to the right in each section of scroll

File: components/BackgroundCircles.tsx
Comment (vvyn) [inline components/BackgroundCircles.tsx]: You may want to delete this commented out code.

File: components/BackgroundCircles.tsx
Comment (vvyn) [inline components/BackgroundCircles.tsx]: These as well.

File: components/homeComponents/HomeHero2.tsx
Comment (vvyn) [inline components/homeComponents/HomeHero2.tsx]: I think the font should be Fredoka One.

File: components/BackgroundCircles.tsx
Comment (vvyn) [reply components/BackgroundCircles.tsx]: I liked the floating NavBar effect though.

**Review by vvyn — DISMISSED** [review]

File: components/AppHeader2/wrapper.tsx
Comment (DedsecKnight) [inline components/AppHeader2/wrapper.tsx:10]: Thoughts on making the navbar sticky on top?

File: components/BackgroundCircles.tsx
Comment (DedsecKnight) [inline components/BackgroundCircles.tsx]: Remove this.

**Review by DedsecKnight — DISMISSED** [review]
Might wanna consider making the navbar sticky on top. Other than that, PR looks good to me and will merge it in once merge conflict are resolved.

File: components/AppHeader2/wrapper.tsx
Comment (doanhtu07) [reply components/AppHeader2/wrapper.tsx:10]: Yeah I can do that. So no more hiding?

**Review by DedsecKnight — DISMISSED** [review]
lgtm! will merge this in once all the sections are linked properly.

**Review by vvyn — DISMISSED** [review]
looks good!

**Review by DedsecKnight — APPROVED** [review]
lgtm! will merge this in now since we already had 2 approvals prior.

**Pull request merged**
